lasagne/layers/conv.py

```diff
@@ -117,23 +109,22 @@ class Conv1DLayer(Layer):
         An integer or a 1-element tuple specifying the stride of the
         convolution operation.
 
-    border_mode : str, one of 'valid', 'full', 'same'
-        A string indicating the convolution border mode.
+    pad : {0, (0,), 'full', 'same'} (default: 0)
```

Comment:
> It would be nice if you could reuse PadLayer,

I'd still be practically reusing its `get_output_for()`, I just wouldn't use its `get_output_shape_for()`, but we've got `conv_output_length()` for that anyway. I think it's a lot cleaner to keep everything within the `Conv2DLayer` and `Conv1DLayer`, especially if we anticipate changing it later. Okay then, thanks for the feedback, will cope with this soon!